_data/tagging-status.yml

```diff
@@ -142,14 +143,14 @@
    type: pkg
    status: currently-unsupported
    comments: See [[1]](#ref1) and also
-   issue: 70.
+   issue: 70
    related-issues: [41]
 
  - name: longtable
    type: pkg
-   status: unknown
+   status: supported
```

Comment:
I think we need "partial-support" as a value (showing yellow)

Reply:
no sure. The question is if supported/unsupported means "we did/can do something" (which is true for quite a lot of package) or "fully works with the tagging code" (which is probably not true for many packages yet ...).

Reply:
In my view supported should mean "fully supported" (until we receive a bug report),  partial-support means there are restrictions and we should list them in the comments. unsupported mean we aren't going to support it and it will not work with tagging (mostly forever) and currently-unsupported means it does not work but we have it in one way or the other on our roadmap

Reply:
I think "currently-unsupported" is then a bit misleading, perhaps "currently clashes"?

Reply:
or currently-breaks  or  currently-incompatible (which is what we call it in the label)

Reply:
> I think we need "partial-support" as a value (showing yellow)

done